_data/rules.yml

```diff
@@ -1532,9 +1532,90 @@ groups:
                 for: 3m
               - name: Nats high routes count
                 description: High number of NATS routes ({{ $value }}) for {{ $labels.instance }}
-                query: "gnatsd_routez_num_routes > 10"
+                query: "gnatsd_varz_routes > 10"
                 severity: warning
                 for: 3m
+              - name: Nats high memory usage
+                description: NATS server memory usage is above 200MB for {{ $labels.instance }}
+                query: "gnatsd_varz_mem > 200000000"
+                severity: warning
+                for: 5m
+              - name: Nats slow consumers
+                description: There are slow consumers in NATS for {{ $labels.instance }}
+                query: "gnatsd_varz_slow_consumers > 0"
+                severity: critical
+                for: 3m
+              - name: Nats server down
+                description: NATS server has been down for more than 5 minutes
+                query: "absent(gnatsd_connz_total)"
```

Comment:
is it applicable when you have multiple NATS servers ?

Reply:
i am using single server setup so I didn't think about this corner case.
We can use the up PromQL exprr instead then.
`up{job="nats"}`

Reply:
Yes, you can find some examples with other exporters